scripts/docs/generate-man.sh

```diff
@@ -4,32 +4,18 @@ set -eu
 
 : "${MD2MAN_VERSION=v2.0.5}"
 
-export GO111MODULE=auto
-
-function clean {
-  rm -rf "$buildir"
+function clean() {
+	rm -f go.mod
```

Comment:
Shouldn't these scripts only delete `go.mod` if it didn't exist when this script started?
(ie, only if this script running `./scripts/vendor init` is what created it)

Reply:
Good point; technically, there "shouldn't" be one, but I know I tend to have one present (as a symlink); let me try how it handles that.